sys/net/network_layer/fib/fib.c

```diff
@@ -522,21 +527,21 @@ void fib_init(fib_entry_t table[])
         prefix_rp[i] = NULL;
     }
 
-    for (size_t i = 0; i < FIB_MAX_FIB_TABLE_ENTRIES; ++i) {
-        table[i].iface_id = 0;
-        table[i].lifetime.seconds = 0;
-        table[i].lifetime.microseconds = 0;
-        table[i].global_flags = 0;
-        table[i].global = NULL;
-        table[i].next_hop_flags = 0;
-        table[i].next_hop = NULL;
+    for (size_t i = 0; i < table->size; ++i) {
```

Comment:
I know this may be out of this PR's scope: but can you replace the initialization loop with a simple memset, while you are at it?